rasa/core/channels/rasa_chat.py

```diff
@@ -27,35 +28,58 @@ def from_credentials(cls, credentials):
 
     def __init__(self, url):
         self.base_url = url
+        self.jwt_key = None
+        self.jwt_algorithm = None
 
-    async def _check_token(self, token):
-        url = "{}/auth/verify".format(self.base_url)
-        headers = {"Authorization": token}
-        logger.debug("Requesting user information from auth server {}.".format(url))
-
+    async def _fetch_public_key(self) -> None:
+        public_key_url = "{}/version".format(self.base_url)
         async with aiohttp.ClientSession() as session:
             async with session.get(
-                url, headers=headers, timeout=DEFAULT_REQUEST_TIMEOUT
+                public_key_url, timeout=DEFAULT_REQUEST_TIMEOUT
             ) as resp:
-                if resp.status == 200:
-                    return await resp.json()
-                else:
-                    logger.info(
-                        "Failed to check token: {}. "
-                        "Content: {}".format(token, await resp.text())
+                if resp.status != 200:
+                    logger.info("Failed to fetch ")
```

Comment:
Can you make the log message more informative? Something like "Failed to fetch public key from 'url'. Response: ...."